hat/assets/js/apps/Iaso/domains/projects/components/FeatureFlagsSwitches.tsx

```diff
@@ -53,17 +53,60 @@ export const FeatureFlagsSwitches: React.FunctionComponent<Props> = ({
         [projectFeatureFlagsValues, handleChange],
     );
 
+    const [hiddenFeatureGroups, setHiddenFeatureGroups] = useState<string[]>(
+        [],
+    );
+
+    const toggleFeatureGroup = useCallback(
+        group => {
+            const indexOfHidden = hiddenFeatureGroups?.indexOf(group) ?? -1;
+            if (indexOfHidden < 0) {
+                setHiddenFeatureGroups([...hiddenFeatureGroups, group]);
+            } else {
+                setHiddenFeatureGroups(
+                    hiddenFeatureGroups.filter((_, i) => i !== indexOfHidden),
+                );
+            }
+        },
+        [hiddenFeatureGroups, setHiddenFeatureGroups],
+    );
+
     const columns = useFeatureFlagColumns(
         setFeatureFlag,
+        toggleFeatureGroup,
         projectFeatureFlagsValues,
     );
 
+    const featureFlagsData = useMemo(() => {
+        const groupRows: string[] = [];
+        // Assuming order from Back end is related to groups as well ..
+        // Maybe we should make sure of that.
```

Comment:
you could write a test in the backend for that 😁

Reply:
and/or make the api accept an `order`queryParam and pass it when fetching data for this page